src/CattyTests/DataModel/VariablesContainerTest.swift

```diff
@@ -236,4 +236,41 @@ final class VariablesContainerTest: XCTestCase {
         XCTAssertTrue(lists[0].isList)
         XCTAssertTrue(lists[1].isList)
     }
+
+    func testIsEqualToVariablesContainer() {
+        let objectA = SpriteObject()
+        objectA.name = "testObjectA"
+
+        let list = UserVariable(name: "testList", isList: true)
+        let variable = UserVariable(name: "testvariable", isList: false)
+
+        let container1 = VariablesContainer()
+        let container2 = VariablesContainer()
+
+        container1.addObjectList(list, for: objectA)
+        container1.addObjectVariable(variable, for: objectA)
+
+        container2.addObjectList(list, for: objectA)
+        container2.addObjectVariable(variable, for: objectA)
+
+        XCTAssertTrue(container1.isEqual(to: container2))
+    }
+
+    func testMutableCopy() {
+        let objectA = SpriteObject()
+        objectA.name = "testObjectA"
+
+        let list = UserVariable(name: "testList", isList: true)
+        let variable = UserVariable(name: "testvariable", isList: false)
+
+        let container = VariablesContainer()
+
+        container.addObjectList(list, for: objectA)
+        container.addObjectVariable(variable, for: objectA)
+
+        let copyContainer = container.mutableCopy() as! VariablesContainer
+
+        XCTAssertTrue(container.isEqual(to: copyContainer))
+        XCTAssertFalse(container == copyContainer)
```

Comment:
Please also check whether the items of the container have the same reference or not.